shared-bindings/rgbmatrix/RGBMatrix.c

```diff
@@ -186,7 +186,22 @@ STATIC void preflight_pins_or_throw(uint8_t clock_pin, uint8_t *rgb_pins, uint8_
 //|         flicker during updates.
 //|
 //|         A RGBMatrix is often used in conjunction with a
-//|         `framebufferio.FramebufferDisplay`."""
+//|         `framebufferio.FramebufferDisplay`.
+//|
+//|         :param int width:  The overall width of the display in pixels
+//|         :param int height: The overall height of all displays in pixels
+//|         :param int tile: In a multi-row display, the number of rows of panels
+//|         :param int bit_depth: The color depth of the display. A value of 1 gives 8 colors, a value of 2 gives 64 colors, and so on. Increasing bit depth increases the CPU and RAM usage of the RGBMatrix, and may lower the panel refresh rate. The framebuffer is always in RGB565 format regardless of the bit depth setting
+//|         :param bool serpentine: In a multi-row display, True when alternate rows are rotated 180°, which can reduce wiring length
+//|         :param Sequence[digitalio.DigitalInOut] rgb_pins: The display's "RGB pins"
+//|         :param Sequence[digitalio.DigitalInOut] addr_pins: The display's "address pins"
+//|         :param digitalio.DigitalInOut clock_pin: The display's "clock pin"
+//|         :param digitalio.DigitalInOut latch_pin: The display's "latch pin"
+//|         :param digitalio.DigitalInOut output_enable_pin: The display's "output enable" pin
```

Comment:
I am not sure why these are in quotes. It's not a "pretend" clock pin, which is sometimes what putting something in quotes means. I think you could drop the quotes. Also I would say "matrix" instead of "display".

Reply:
fixed! & made some additional updates